plug.vim

```diff
@@ -1411,7 +1425,7 @@ while 1 " Without TCO, Vim stack is bound to explode
     endif
   else
     call s:spawn(name,
-          \ printf('git clone %s %s %s %s 2>&1',
+          \ printf('git clone %s %s %s %s',
           \ has_tag ? '' : s:clone_opt,
```

Comment:
`s:clone_opt` is the only blocker for using list type for the command passed to `s:spawn`. Using list type command lets Vim/Neovim run git directly without the shell. I'll refactor `s:clone_opt` to a list in separate PR since it affects the Python and Ruby code.